charts/node-analyzer/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## Change Log
 
 This file documents all notable changes to Sysdig Node Analyzer Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+## 1.7.31
+### Minor changes:
+* BenchmarkRunner
+    * bump to version 1.1.0.2
+        * bump linux bench to 1.1.0.1
```

Comment:
from this PR i cannot see anything bumped _to_ version `1.1.0.1`, only from `1.1.0.1` to `1.1.0.2`. does this entry, and the one in the other changelog, need to be adjusted?

Reply:
yup could use better wording.

Reply:
the versions, as strange as they may seems, are correct. I'll update to remove the versions that reference embedded components.